src/modules/loadScore.js

```diff
@@ -0,0 +1,25 @@
+import apiURL from './api.js';
+
+const scoreList = document.getElementById('scoreList');
+
+export default async function loadScore() {
+  await fetch(apiURL)
+    .then((response) => response.json())
+    .then((json) => {
```

Comment:
- [ ]  You should use `async` and `await` with the `fetch` method to send and receive data.

Reply:
@ishimwezachee Thank you for the review, it would be a great help if you could guide me through this. I am confused about the implementation.